docs/v3/how-to-guides/automations/chaining-deployments-with-events.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: How to chain workflow runs with events
```

Comment:
Do we know the common way users express this? Maybe something like "Create dependencies between workflows with events"